lib/membrane/rtp/ssrc_router.ex

```diff
@@ -295,6 +309,19 @@ defmodule Membrane.RTP.SSRCRouter do
     )
   end
 
+  defp emit_inbound_frame_event(actions, ctx) do
+    for {:buffer, {pad, %Buffer{} = buffer}} <- actions,
+        buffer.metadata.rtp.marker,
+        Map.has_key?(ctx.pads, pad) do
```

Comment:
Instead of checking againts pad in ctx.pads, I would emit event every time we receive some buffer. We could also get rid of `emit_inbound_frame_event` in `handle_pad_added(:output)`.

Reply:
This would make it inconsistent with "inbound-rtp.packets". Effectively, you could include a frame in the metrics even though you didn't count packets that make it up.